### ISSUE TITLE
Tooltip alignment

### ISSUE THREAD

**Issue by zepfietje** [laravel-filament/filament#13335]
Tooltip should align relative to actual table cell or text entry content.
This has been brought up before, so let's look into ways to fix this.

<img width="366" alt="Screenshot 2024-06-20 at 11 26 11" src="https://github.com/filamentphp/filament/assets/44533235/8aedcfd5-a0e8-4793-95bd-1df6dbe2618f">

**Comment by zepfietje** [comment]
Do you think this should be v4 instead, @danharrin?

**Comment by danharrin** [comment]
I don't think so?

**Comment by zepfietje** [comment]
Cool, seems like a bug fix to me too.

**Comment by colorants** [comment]
Hi, the problem seems to exist still. Are there any hacky ways to fix it or should I just ignore it for now.

**Comment by ShaungBhone** [comment]
I've already fixed that, but we need to move the tooltip to `text-column.blade.php` instead of the entire `column.blade.php`. I'm not sure if you guys will accept this change or not.

![Image](https://github.com/user-attachments/assets/8f35b175-14af-45e1-8638-9cc3f408dd82)

**Comment by jsheffers** [comment]
Would something like this work? Attaching the x-on:click to a child element?

Concept would be to split the attributes to container & inner attributes. 

<img width="2810" height="1350" alt="Image" src="https://github.com/user-attachments/assets/dc3e58fa-6d64-4460-aee3-d2323eeaab5d" />
<img width="922" height="86" alt="Image" src="https://github.com/user-attachments/assets/3f5853ed-1717-4144-9744-dbe3828f4bdc" />

<img width="1360" height="596" alt="Image" src="https://github.com/user-attachments/assets/026d11f6-12ca-4237-9773-621425b115a6" />


Styling updates
<img width="494" height="258" alt="Image" src="https://github.com/user-attachments/assets/05149560-ab1a-42a5-bb04-2dc479c797c2" />

**Comment by jsheffers** [comment]
@danharrin any thoughts on this approach?

**Comment by danharrin** [comment]
Does it work when there are multiple items (array state) in the column?

**Comment by jsheffers** [comment]
Submitted a pull request 👆🏻

**Comment by jsheffers** [comment]
> Does it work when there are multiple items (array state) in the column?

Yes, it does. I've attached a screen recording to the pull request.
